api/service/debug.go

```diff
@@ -25,4 +25,27 @@ func (s *Service) addDebugEndpoints() {
 			return
 		},
 	)
+
+	s.Router.Methods(http.MethodPost).Path("/getProject").HandlerFunc(
```

Comment:
http.MethodGet with body is fine. call the path /project. and also move to a new `project.go`